org.eclipse.xtext.xtext.ui.examples/projects/org.eclipse.xtext.example.domainmodel/build.properties

```diff
@@ -12,4 +12,7 @@ additional.bundles = org.eclipse.xtext.xbase,\
                      org.eclipse.xtext.xtext.generator,\
                      org.eclipse.emf.codegen.ecore,\
                      org.eclipse.emf.mwe.utils,\
-                     org.eclipse.emf.mwe2.launch
+                     org.eclipse.emf.mwe2.launch,\
+                     org.apache.log4j,\
+                     org.apache.commons.logging,\
+                     com.ibm.icu
```

Comment:
But why should there be additional dependencies which are already listed as required bundle? The bundles listed there should only complete the runtime classpath for the Xtext generator with bundles that aren't on the classpath by Require-Bundle or Import-Package.

The actual list of additional bundles that are sufficient to execute the generator are:
```
additional.bundles = org.eclipse.xtext.xtext.generator,\
                     org.eclipse.emf.codegen.ecore,\
                     org.eclipse.emf.mwe.utils,\
                     org.eclipse.emf.mwe2.launch,\
                     org.eclipse.emf.mwe2.lib,\
                     org.apache.commons.logging,\
                     org.apache.log4j,\
                     com.ibm.icu
```

So I think this is rather a problem of the wizard. It should not add the dependencies
- org.eclipse.xtext.xbase
- org.eclipse.xtext.common.types
- org.objectweb.asm
as additional bundle.

If you create a new project, remove those dependencies and run the Xtext generator, you should see that it succeeds and no compile error exists. (Actually I get a compile error in .tests, but this seems to be another issue that I'll raise now).

Do you agree that I should raise an issue for the wizard?